src/organization/actions/register-organization.action.ts

```diff
@@ -7,14 +8,29 @@ export const registerOrganization = async (
   fetcher: Fetcher,
   request: IRegisterOrganizationRequest,
 ): Promise<IOrganizationResponse> => {
-  const response = await fetcher.post<IOrganizationResponse>(
-    endpoints.REGISTER_ORGANIZATION,
-    request,
-  );
-
-  if (response.status !== 201) {
-    throw errorFactory.create(OrganizationErr.REGISTER_ORGANIZATION_FAILED);
+  try {
+    const response = await fetcher.post<IOrganizationResponse>(
+      endpoints.REGISTER_ORGANIZATION,
+      request,
+    );
+    switch (response.status) {
+      case 201:
+        return response.data;
+      case 403:
+        throw errorFactory.create(
+          OrganizationErr.REGISTER_ORGANIZATION_TO_PERSON_FORBIDDEN,
```

Comment:
```suggestion
          OrganizationErr.REGISTER_ORGANIZATION_FORBIDDEN,
```

Cette route est différente de l'autre route "register organization to person".